### PR TITLE
Fix automated tests for about:history page

### LINKED ISSUE #8524: About history failing tests
**Describe the issue you encountered:**

This tests are failing for me locally.

Steps:
1) `npm run watch-test`
2) `TEST_DIR=about npm run testsuite`

1) about:history page content does NOT use customTitle when displaying entries:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  2) about:history page content defaults to sorting table by time DESC:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  3) about:history page content "after all" hook:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  4) about:history multi-select behavior selects multiple rows when clicked with cmd/control:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  5) about:history multi-select behavior selects multiple contiguous rows when shift clicked:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  6) about:history multi-select behavior deselects everything if something other than the table is clicked:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  7) about:history multi-select behavior does not lose selection if table is sorted:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.
  

  8) about:history multi-select behavior "after all" hook:
     Error: timeout of 30000ms exceeded. Ensure the done() callback is being called in this test.

**Additional info**
We notice some lag as well on ledger. I am thinking that about pages has some lag. Maybe related to  this commit https://github.com/brave/browser-laptop/commit/7168987205d567c297470a175038af4e03cc1ded. I notice that when opening history tab CPU spikes and interaction is laggy (hover has a big delay, click as well). @bridiver what do you think?

### PR DESCRIPTION
**Pull request by romamatusevich**
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:
Run `mocha test/about/historyTest.js`

Close #8524

### CODE REVIEW

File: test/about/historyTest.js [hunk 1]
```diff
@@ -132,14 +136,12 @@ describe('about:history', function () {
...
       yield this.app.client
-        .tabByUrl(aboutHistoryUrl)
-        .loadUrl(aboutHistoryUrl)
         .click('table.sortableTable td.title[data-sort="Brave"]')
         .keys(Brave.keys.SHIFT)
         .click('table.sortableTable td.title[data-sort="https://www.youtube.com"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="Brave"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://www.youtube.com"]')
-        .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://brave.com/test"]')
+        .waitForVisible('table.sortableTable td.title[data-sort="https://brave.com/test"]')
```
Comment (NejcZdovc) [inline test/about/historyTest.js]: Can you please explain why `selected` should be removed? I just run `npm run test -- --grep="about:history"` on a local machine and it's working correctly. Also master is not failing. Same goes for all `selected` removed bellow.
Thank you

Comment (romamatusevich) [inline test/about/historyTest.js]: We have sites to test:
- { location: 'https://brave.com', title: 'Brave' },
- { location: 'https://brave.com/test', customTitle: 'customTest' },
- { location: 'https://www.youtube.com' },
- { location: 'https://www.facebook.com' }

Per this test scenario:
- click on [data-sort="Brave"]
- activate shift and click on [data-sort="https://www.youtube.com"]

Selected should be:
- { location: 'https://brave.com', title: 'Brave' }
- { location: 'https://www.youtube.com' }

Not selected should be:
- { location: 'https://brave.com/test', customTitle: 'customTest' }
- { location: 'https://www.facebook.com' }

Comment (romamatusevich) [inline test/about/historyTest.js]: By the way I see that in TraviCI tests for about:history are passed.

Tried locally, but get errors:

1) about:history page content "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19

  2) about:history double click behavior "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19

  3) about:history multi-select behavior "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19

Comment (NejcZdovc) [inline test/about/historyTest.js]: You should reinstall brave (https://gist.github.com/NejcZdovc/5dd0e6789049c2ff715ef891c87f9760) and update node to version 7.9. This error should be fixed then.

Comment (romamatusevich) [inline test/about/historyTest.js]: Unfortunately after the cleanup and install the same result with errors. Anyway this issue is specific for me so I close the PR.

**Pull request closed without merging**